services/enterprise/service.go

```diff
@@ -0,0 +1,241 @@
+package enterprise
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/influxdata/enterprise-client/v2"
+	"github.com/influxdata/enterprise-client/v2/admin"
+	"github.com/influxdb/influxdb/monitor"
+)
+
+// Service represents the enterprise service.
+type Service struct {
+	MetaStore interface {
+		ClusterID() uint64
+		NodeID() uint64
+	}
+	Monitor interface {
+		Statistics(tags map[string]string) ([]*monitor.Statistic, error)
+		RegisterDiagnosticsClient(name string, client monitor.DiagsClient)
+	}
+
+	enabled       bool
+	hosts         []*client.Host
+	statsInterval time.Duration
+	version       string
+	mu            sync.Mutex
+	lastContact   time.Time
+	adminPort     string
```

Comment:
Can you add a space above `adminPort` and below `version`, to indicate that `mu` is protecting `lastContact`?